src/style.css

```diff
@@ -1,3 +1,66 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  height: 60px;
+}
+
```

Comment:
please add .logo and give it size => width and height should be 40px
if your image will be bigger or smaller, your header will crush without those styles